platforms/gcp-dataproc-classical-airgapped/spark_healthcare_nlp_classical.py

```
@@ -0,0 +1,77 @@
+chunkerMapper = ChunkMapperModel.load("gs://spark-healthcare-nlp/models/drug_brandname_ndc_mapper_en")\
+      .setInputCols(["chunk"])\
+      .setOutputCol("ndc")\
+      .setRels(["Strength_NDC"])
+
+pipeline = Pipeline().setStages([document_assembler,
+                                 chunkerMapper])  
+
+model = pipeline.fit(spark.createDataFrame([['']]).toDF('text')) 
+
+print("******************** Pipeline ready! ********************* ")
+lp = LightPipeline(model)
```

Ruff (0.11.9)

`LightPipeline` may be undefined, or defined from star imports

(F405)